### DISCUSSION TITLE
Run Milvus on M1 Apple Silicon

### DISCUSSION THREAD

**Discussion by sgaseretto** [milvus-io/milvus#4631] [Q&A and General discussion]
Has anyone tried to run the Milvus Container on Apple Silicon's M1. Did it work? I'm asking since the docs says [this.](https://milvus.io/docs/v0.10.5/product_faq.md?Does-Milvus-support-non-x86-architecture?)

**Comment by shiyu22** [comment]
Hi, does your machine support docker? You can install Milvus as long as you have a Docker environment set up on your operating system.

**Comment by gujun720** [comment]
I answered a similar issue on Slack. Let me paste here also.

On the data center side, Milvus is still focusing on X86 processors. Milvus 0.6 could support the Arm processor, but with limited functions, because it is for the edge side.
So the question is if there is any plan for supporting Arm at the data center side. So far there is no. because the data center Arm processor is not common...

It's hard for the community to find a standard Arm processor.
